utility_analysis_new/dp_engine.py

```diff
@@ -38,8 +41,14 @@ def aggregate(self,
         self._is_public_partitions = public_partitions is not None
         result = super().aggregate(col, params, data_extractors,
                                    public_partitions)
+        error_aggregators = self._create_aggregate_error_compound_combiner(
+            params)
         self._is_public_partitions = None
-        return result
+        keyed_by_same_key = self._backend.map(
```

Comment:
you need to map Metrics to AggregatedAccumulator with error_aggregators.create_accumulator